app/components/ui/atoms/PersonPod.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+import { Action, Button } from '@pubsweet/ui'
+import { Flex, Box } from 'grid-styled'
+
+import Icon from './Icon'
+
+const RegularP = styled.p`
+  font-size: ${th('fontSizeBase')};
+  line-height: ${th('fontLineHeightBase')};
+  margin: 0;
+`
+
+const SmallP = styled.p`
+  font-size: ${th('fontSizeBaseSmall')};
+  line-height: ${th('fontLineHeightBaseSmall')};
+  // vertical spacing of 6px comes from: 24px grid - fontLineHeightBaseSmall
+  margin: 3px 0;
+`
+
+const SmallAction = styled(Action)`
+  font-size: ${th('fontSizeBaseSmall')};
+  line-height: ${th('fontLineHeightBaseSmall')};
+  margin: 3px 0;
+`
+
+const StyledButton = styled(Button)`
+  height: 100%;
+  border: none;
+`
+
+const StyledPicker = styled(Flex)`
+  display: flex;
+  justify-content: space-between;
+
+  background: ${th('colorSecondary')};
+  border: ${th('borderWidth')} ${th('borderStyle')} ${th('colorBorder')};
+  border-radius: ${th('borderRadius')};
+  color: ${th('colorText')};
+  font-family: ${th('fontInterface')};
+  font-size: ${th('fontSizeBase')};
+  min-width: 312px;
+  height: 128px;
+`
+
+const PersonPod = ({ onIconClick, textContainer, icon, ...props }) => (
+  <StyledPicker>
+    {textContainer}
+    <Flex flexDirection="column" justifyContent="center">
+      <StyledButton onClick={onIconClick}>{icon}</StyledButton>
+    </Flex>
+  </StyledPicker>
+)
+
+const PlusIcon = <Icon size={3}>Plus</Icon>
+const RubbishBinIcon = <Icon size={3}>RubbishBin</Icon>
+const SelectedTickIcon = <Icon size={4}>SelectedTick</Icon>
+
+const ChangingIcon = iconState => {
+  switch (iconState) {
+    case 'add':
+      return PlusIcon
+    case 'remove':
+      return RubbishBinIcon
+    case 'selected':
+      return SelectedTickIcon
+    default:
+      return PlusIcon
+  }
+}
+
+const ChosenPerson = ({
+  name,
+  institution,
+  keywords,
+  isKeywordClickable,
+  onKeywordClick,
+  isStatusShown,
+  status,
+  ...props
+}) => (
+  <Box m={2}>
+    <RegularP>{name}</RegularP>
+    <RegularP>{institution}</RegularP>
+    {isKeywordClickable && (
+      <SmallAction onClick={onKeywordClick}>{keywords}</SmallAction>
+    )}
+    {!isKeywordClickable && <RegularP>{keywords}</RegularP>}
+    {isStatusShown && <SmallP>{status}</SmallP>}
+  </Box>
+)
+
+const ChosenPersonPod = props => (
+  <PersonPod
+    icon={ChangingIcon(props.iconState)}
+    textContainer={ChosenPerson(props)}
```

Comment:
I find this a bit confusing where ChangingIcon and ChosenPerson are defined as components and start with an uppercase letter, but are used as functions. In this case it makes no functional difference but not all functions are components and not all components are functions.

Reply:
I'd vote for `ChangingIcon` -> `podIcon`
`ChosenPerson` -> `podPerson`

Reply:
[Pod people!](http://www.darkcrystal.com/galleries_podpeople.php)

![image](https://user-images.githubusercontent.com/115310/42993476-60fecfca-8c03-11e8-89fa-7c13656e2098.png)

Reply:
Are you thinking ..... Easter Egg?

Reply:
Hahahahahahaa, pod people. These films are trippy!

@diversemix So funny that you would say that because `PodIcon` was actually the original name that I gave to it! But I changed it to `ChangingIcon` because using a verb rather than a noun felt like it better described what this bit of code is doing.

I think the lack of clarity in the code is a reflection of the confusion in my brain as I was building it. So thanks so much for this comment @tamlyn, it really helped me pinpoint where my own confusion was coming from! The reason that a verb felt more appropriate is because it's a function rather than a component.

But I was still thinking about `ChangingIcon`/`PodIcon` as though it were a component, rather than a function that creates a component. If you guys could take a look at my latest commit & let me know what you think of these names instead, that would be great :grin:

Reply:
@jsms90 Naming things is always difficult! Looking now.